### ISSUE TITLE
After dispalying On device files only, All files are not shown anymore

### ISSUE THREAD

**Issue by avalx** [nextcloud/android#5908]
### Steps to reproduce
1. Open Nextcloud client
2. Sidebar -> On device = only dirs/files on device are shown
3. Sidebar - All files = still only dirs/files on device are shown

### Expected behaviour
After 3. all dirs/files should be visible

### Actual behaviour
Even if I choose All files, only the dirs/files on my device are shown.

### Can you reproduce this problem on https://try.nextcloud.com?
I didn't try this yet but I've tried on two different accounts: one on my local NC server 18.0.3 and one on Disroot.org.

### Environment data
Android version: 9

Device model: Oneplus 3, Samsung Tab A WiFi

Stock or customized system: stock ROM, Root

Nextcloud app version: 3.11.0

Nextcloud server version: 18.0.3 on my local serve, not sure for Disroot.org

Reverse proxy: 

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by avalx** [comment]
Additional info!

I tried some different options in the sidebar, like Favorites, Photos, Shared. They are all dispaying as expected, for example photos are shown OK. After that I again chose All files and still only On device files are shown.

But when I choose Activities (they are displayed as expected) and then All files again - voilà all directories/files are shown!

**Comment by tobiasKaminsky** [comment]
Fixed via: #5617
